requirements/edx/github.txt

```diff
@@ -53,7 +53,7 @@ git+https://github.com/edx/nltk.git@2.0.6#egg=nltk==2.0.6
 -e git+https://github.com/appliedsec/pygeoip.git@95e69341cebf5a6a9fbf7c4f5439d458898bdc3b#egg=pygeoip
 -e git+https://github.com/jazkarta/edx-jsme.git@c5bfa5d361d6685d8c643838fc0055c25f8b7999#egg=edx-jsme
 git+https://github.com/edx/django-pyfs.git@1.0.3#egg=django-pyfs==1.0.3
-git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a0c695#egg=django-cas
+git+https://github.com/mitocw/django-cas.git@47cfef16cbbf305f943cceb8c0b02514194eb9fa#egg=django-cas
```

Comment:
Yes, I've created a release tag to match the version and I've just updated this PR: v2.1.1